static/data/meetings/notes/2023-07-20/index.md

```diff
@@ -0,0 +1,277 @@
+# Podman Community Cabal Meeting Notes
+
+Attendees: Aditya Rajan, Anders F Björklund, Ashley Cui, Ed Santiago Munoz, Jake Correnti, Justin Jereza, Lokesh Mandvekar, Martin Jackson, Matt Heon, Miloslav Trmac, Mohan Boddu, Nalin Dahyabhai, Paul Holzinger, Tom Sweeney, Valentin Rothberg
+
+## July 20, 2023 Topics
```

Comment:
@TomSweeneyRedHat I see the format is somewhat different here than other notes.
e.g. There is a line like
```
June 15, 2023 11:00 a.m. Eastern (UTC-5)
```
Which is used by the code to fetch the date field.
I think a line as above needs to be added here.